config.js

```diff
@@ -215,6 +218,9 @@ module.exports = {
     DEFAULT_PERSIST_REDIS_SAMPLE_STORE_MILLISECONDS,
   port,
   prioritizeJobsFrom,
+  queueStatsActivityLogsInterval,
+  queueTimeMillis95th: pe.QUEUESTATS_95TH_WARNING_MILLIS ||
+    undefined,
   rateLimit,
```

Comment:
If undefined, we don't treat any as warning; but I think @shriramshankar's comment is just that you don't need to add " || undefined" here.